src/gatsby-theme-carbon/components/LeftNav/ResourceLinks.js

```diff
@@ -4,7 +4,7 @@ import ResourceLinks from 'gatsby-theme-carbon/src/components/LeftNav/ResourceLi
 const links = [
   {
     title: 'IDL library',
```

Comment:
```suggestion
    title: 'IDL Sketch library',
```
We should indicate these are sketch links as they won't work if the user doesn't have sketch installed.